parametric/apps/golang/main.go

```diff
@@ -66,9 +75,13 @@ func (s *apmClientServer) InjectHeaders(ctx context.Context, args *InjectHeaders
 	if err != nil {
 		fmt.Println("error while injecting")
 	}
-	distr := map[string]string{}
+	distr := []*HeaderTuple{}
 	for k, v := range headers {
-		distr[k] = v
+		var headerTuple = &HeaderTuple{
+			Key: k,
+			Value: v,
+		}
+		distr = append(distr, headerTuple)
```

Comment:
```suggestion
		distr = append(distr, &HeaderTuple{Key: k, Value: v})
```

Reply:
Fixed in b843e78cfd5be0963f4f60c1c7af5ac01f9360fc